### PR TITLE
Trichromat Modification Trait

### PR DESCRIPTION
**Pull request by VMSolidus**
# Description

This PR brings back a previous briefly existing feature, that of the NormalVisionComponent, which now exists as a single point positive trait that can be taken only by Harpies and Vulpkanin(A list to later be expanded if we ever get new species that have modified vision types). This trait is called "Trichromat Modification", and it simply removes any species based vision modifications an entity may have, such as Ultraviolet Vision. 

:cl:
- add: Trichromat Modification has been added as a new positive trait. It can be taken by Harpies and Vulpkanin to buy off their unique vision negative traits.

### CODE REVIEW

File: Resources/Prototypes/Traits/neutral.yml
Comment (DangerRevolution) [inline Resources/Prototypes/Traits/neutral.yml:1]: Can we see this in-game?

Comment (VMSolidus) [reply Resources/Prototypes/Traits/neutral.yml:1]: Literally nothing to show, it just removes the traits, making you see in RGB.

**Review by DangerRevolution — APPROVED** [review]

**Review by DangerRevolution — APPROVED** [review]
This should be reviewed before Saturday please :))

**Comment by DangerRevolution** [comment]
Also @VMSolidus can you rename the fork-based options from Delta-V since you're already touching the vision removing?

**Review by Peptide90 — APPROVED** [review]

**Pull request merged**